frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -637,18 +638,15 @@ Except when in two columns mode, where this is limited to showing only the previ
         })
         table.insert(menu_items.long_press.sub_item_table, {
             text_func = function()
-                if G_reader_settings:readSetting("highlight_non_touch_interval") == 1 then
-                    return T(_("Interval to speed-up rate: %1 second"), G_reader_settings:readSetting("highlight_non_touch_interval", 1))
-                else
-                    return T(_("Interval to speed-up rate: %1 seconds"), G_reader_settings:readSetting("highlight_non_touch_interval", 1))
-                end
+                local highlight_non_touch_interval = G_reader_settings:readSetting("highlight_non_touch_interval") or 1
+                return T(NC_("Speed-up rate interval: %1 second", "Speed-up rate interval: %1 seconds", highlight_non_touch_interval), highlight_non_touch_interval)
```

Comment:
Incorrect call, `NC_` requires context.

Reply:
Doesn't `NC` stand for "no context"? ;) 

> requires context

Joke aside, what does this mean?

Reply:
My bad, just N_() 
https://koreader.rocks/doc/modules/gettext.html#ngettext